TDD_Training/FindMultiplesTests/FindMultiplesTests.cs

```diff
@@ -0,0 +1,90 @@
+using System;
+using TDD_Training;
+using Xunit;
+
+namespace FindMultiplesTests
+{
+    public class FindMultiplesTests
+    {
+        private readonly FizzBuzzService _findMultiples = new FizzBuzzService();
+
+        [Fact]
+        public void CheckGivenNumber_IfMultipleOfThree_ReturnsFizz()
+        {
+            //Arrange
+            var value = 312;
+            var expected = "Fizz";
+
+            //Act
+            var actual = _findMultiples.isMultipleOf(value);
+
+            //Assert
+            Assert.Equal(expected, actual);
+        }
+
+        [Theory]
+        [InlineData(6)]
+        [InlineData(93)]
+        [InlineData(108)]
+        public void CheckNumber_IfMultipleOfThree_ReturnsFizz(int value)
```

Comment:
Method description should be declarative
something like
Given_when_then
GivenNumberInMultiplesOfThree_IsMultipleOf_ReturnsFizz()